### PR TITLE
Improved BlankLoadScreen performance

### PR DESCRIPTION
**Pull request by Mailaender**
by simply drawing it only once instead of with max FPS.

### CODE REVIEW

File: OpenRA.Mods.Common/LoadScreens/BlankLoadScreen.cs [hunk 1]
```diff
@@ -29,14 +29,18 @@ public virtual void Init(ModData modData, Dictionary<string, string> info)
...
 		{
-			if (Game.Renderer == null)
+			if (Game.Renderer == null || initialized)
 				return;
 
 			// Draw a black screen
 			Game.Renderer.BeginUI();
 			Game.Renderer.EndFrame(new NullInputHandler());
+
+			initialized = true; // PERF: draw the screen only once
```
Comment (abcdefg30) [inline OpenRA.Mods.Common/LoadScreens/BlankLoadScreen.cs]: Style nit: Comments should go on their own line above the line commented on.

**Review by abcdefg30 — COMMENTED** [review]
Lgtm otherwise.

File: OpenRA.Mods.Common/LoadScreens/BlankLoadScreen.cs (hunk 1 above)
Comment (Mailaender) [reply OpenRA.Mods.Common/LoadScreens/BlankLoadScreen.cs]: Moved.

File: OpenRA.Mods.Common/LoadScreens/BlankLoadScreen.cs [hunk 2]
```diff
@@ -29,14 +29,19 @@ public virtual void Init(ModData modData, Dictionary<string, string> info)
 			ModData = modData;
 		}
 
+		bool initialized;
```
Comment (pchote) [inline OpenRA.Mods.Common/LoadScreens/BlankLoadScreen.cs]: Can you please define this with the other fields at the top of the class?

Comment (Mailaender) [reply OpenRA.Mods.Common/LoadScreens/BlankLoadScreen.cs]: Okay.

**Review by pchote — APPROVED** [review]

**Comment by pchote** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/92f792de89f56055535deecf8f5adbdd6cc1ff97

**Pull request merged**